### ISSUE TITLE
Exception with basic client due to coffeescript typo

### ISSUE THREAD

**Issue by focusaurus** [devktor/jsonrpc-node#3]
With a basic client like this:

``` js
const Client = require('jsonrpc-node').TCP.Client;
const client = new Client(9080, 'localhost');
client.call('isEven', {number: 2}, function (error, reply) {
  console.log(error, reply)
});
```

The code crashes with this exception

```
events.js:251
    throw new TypeError('listener must be a function');
    ^

TypeError: listener must be a function
    at Client.once (events.js:251:11)
    at Client.onceConnected (/Users/plyons/projects/learn/jsonrpc-node/node_modules/jsonrpc-node/lib/tcp_client.js:120:21)
    at Client.call (/Users/plyons/projects/learn/jsonrpc-node/node_modules/jsonrpc-node/lib/tcp_client.js:133:23)
    at Object.<anonymous> (/Users/plyons/projects/learn/jsonrpc-node/client.js:3:8)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.runMain [as _onTimeout] (module.js:441:10)
    at Timer.listOnTimeout (timers.js:92:15)
```

I believe this is due to [this line](https://github.com/devktor/jsonrpc-node/blob/f4ead68958aed2c18174cef6792a3db045d68b2c/src/tcp_client.coffee#L70) missing a fat arrow expression `(err)=>` (like the line 2 below it has).

**Comment by devktor** [comment]
fixed in [1427725](https://github.com/devktor/jsonrpc-node/commit/1427725bed0947637997d036452aa94775c70feb), can you confirm?

**Comment by focusaurus** [comment]
Yes, that avoids the error. Thanks!
